### DISCUSSION TITLE
Support for continued TTS model training

### DISCUSSION THREAD

**Discussion by oytunturk** [NVIDIA-NeMo/NeMo#3379] [Q&A] [answered]
Does the TTS training scripts support continuing training from where it left off or initializing a new training session using an older checkpoint's model? I'm specifically asking this for Tacotron2 training but it would be useful to know for any TTS related model training in general.

**Comment by godspirit00** [comment, marked as answer]
See [here](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/core/core.html#experiment-manager).
